scripts/Muon/GUI/Common/contexts/muon_context.py

```diff
@@ -475,14 +474,12 @@ def last_good_data(self, run):
                 return self.gui_context['LastGoodData']
 
     def dead_time_table(self, run):
-        if self.gui_context['DeadTimeSource'] == 'FromADS':
-            return self.gui_context['DeadTimeTable']
-        elif self.gui_context['DeadTimeSource'] == 'FromFile':
+        if self._corrections_context.dead_time_source == "FromFile":
```

Comment:
Do we need this check here? Could we not just do 
```return self._corrections_context.current_dead_time_table_name()``` instead and it will handle if it is fromFile or ADS?